src/Mod/Sketcher/Gui/DrawSketchHandler.h

```diff
@@ -203,6 +205,8 @@ class SketcherGuiExport DrawSketchHandler
 
     Sketcher::SketchObject * getSketchObject();
 
+    void setAngleSnapping(bool val, Base::Vector2d ref = Base::Vector2d(0., 0.));

```

Comment:
@PaddleStroke 

This is actually a very good advice. 

Parameters, as well as variables, should have very meaningful names, so that the user of the code can easily understand what is intended. A code reader should not start thinking... wait... which val? ..oh it is a boolean... yes but what is meant? The "ref" one is actually not better, is it a vector direction what is intended as reference or a point?

These changes you can bring when the review is finished. 

More important is that you try to think about it when writing the code, or at the very least, while reading your own code before PR-ing. That is what we are aiming at.